Documentation/building/cross-platform-testing.md

```diff
@@ -24,13 +24,13 @@ instructions assume you are building for Linux, but are easily modifiable for OS
    respect to object sizes and layout so you need to ensure you have either a
    release coreclr and release System.Private.Corelib or debug coreclr and debug System.Private.Corelib.
 3. A Linux build of CoreFX. We currently have experimental support for building
-   CoreFX on Linux via `build.sh`. 
+   CoreFX on Linux via `build.sh`.
    The other option is:
 
-   * Build the managed parts of CoreFX on Windows. To do so run `build-managed.cmd -os=Linux`. It is okay to build a Debug version of CoreFX and run it
+   * Build the managed parts of CoreFX on Windows. To do so run `build.cmd /p:BuildNative=false /p:RuntimeOS-os=Linux`. It is okay to build a Debug version of CoreFX and run it
```

Comment:
`/p:RuntimeOS-os` seems wrong. Should it be: `/p:RuntimeOS=Linux`?

Reply:
Actually I added the `-os` option in https://github.com/dotnet/corefx/pull/32789 which actually maps to OSGroup not RuntimeOS. I think I was mid-edit when I decided to add the alias.